src/components/StarRating/StarRating.ts

```diff
@@ -0,0 +1,63 @@
+import { $$, Utils, Component, Initialization, ComponentOptions, IComponentBindings } from 'coveo-search-ui';
+import { star as starIcon } from '../../utils/icons';
+
+export interface IStarRatingOptions {
```

Comment:
Should we support different "range" of score ? Let's say somebody has a 0-10 scoring system that we could map back to a five star system ?

Reply:
Done

Reply:
It rounds down though. Could try to have the stars fill in partially with some fancy css but I don't think that's needed for now.